src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,103 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: getContainerWidth($card-width, $card-gap, $card-horizontal-padding);
+  padding: 50px $card-horizontal-padding;
+
+  &--centered {
+    margin: 0 auto;
+  }
+
+  &__card {
+    margin-right: $card-gap;
+    margin-bottom: 46px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &__picture {
+    display: block;
+    margin: 0 auto 40px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__review-container {
+    @include flex-aligning;
+    justify-content: space-between;
```

Comment:
looks like when you use this mixin you always add the `justify-content: center;` rule. Do you think it would be reasonable to add it to the mixin as well?